Reactions/ReactorBDF.cpp

```diff
@@ -65,6 +65,28 @@ get_bdf_matrix_and_rhs(
   get_rho_and_massfracs(soln, rho, massfrac);
 
   eos.RTY2JAC(rho, soln[NUM_SPECIES], massfrac, Jmat1d, consP);
+#ifdef PELE_ROLL_JAC
```

Comment:
@jrood-nrel should we just use if conditions? feels more natural and removes a compile option.